src/steps/big-table/converters.ts

```diff
@@ -0,0 +1,222 @@
+        ),
+      },
+    },
+  });
+}
+
+export function createAppProfileEntity({
+  appProfile,
+  projectId,
+  instanceId,
+}: {
+  appProfile: bigtableadmin_v2.Schema$AppProfile;
+  projectId: string | undefined | null;
+  instanceId: string | undefined | null;
+}) {
+  const instanceName = getLastUrlPart(instanceId!);
+  const appProfileName = getLastUrlPart(appProfile.name!);
+
+  return createGoogleCloudIntegrationEntity(appProfile, {
+    entityData: {
+      source: appProfile,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_APP_PROFILE,
+        _type: ENTITY_TYPE_BIG_TABLE_APP_PROFILE,
+        _key: getAppProfileKey(appProfile),
+        name: appProfile.name,
+        instanceId,
+        etag: appProfile.etag,
+        description: appProfile.description,
+        webLink: getGoogleCloudConsoleWebLink(
+          `/bigtable/instances/${instanceName}/app-profiles/${appProfileName}?project=${projectId}`,
+        ),
+      },
+    },
+  });
+}
+
+export function createClusterEntity({
+  cluster,
+  projectId,
+  instanceId,
+}: {
+  cluster: bigtableadmin_v2.Schema$Cluster;
+  projectId: string | undefined | null;
+  instanceId: string | undefined | null;
+}) {
+  return createGoogleCloudIntegrationEntity(cluster, {
+    entityData: {
+      source: cluster,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_CLUSTER,
+        _type: ENTITY_TYPE_BIG_TABLE_CLUSTER,
+        _key: getClusterKey(cluster),
+        name: cluster.name,
+        instanceId,
+        state: cluster.state,
+        serveNodes: cluster.serveNodes,
+        defaultStorageType: cluster.defaultStorageType,
+        location: cluster.location,
+        kmsKeyName: cluster.encryptionConfig?.kmsKeyName,
+        webLink: getGoogleCloudConsoleWebLink(
+          `/bigtable/instances/${instanceId}/overview?project=${projectId}`,
+        ),
+      },
+    },
+  });
+}
+
+export function createBackupEntity({
+  backup,
+  projectId,
+  instanceId,
+  clusterId,
+}: {
+  backup: bigtableadmin_v2.Schema$Backup;
+  projectId: string | undefined | null;
+  instanceId: string | undefined | null;
+  clusterId: string | undefined | null;
+}) {
+  return createGoogleCloudIntegrationEntity(backup, {
+    entityData: {
+      source: backup,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_BACKUP,
+        _type: ENTITY_TYPE_BIG_TABLE_BACKUP,
+        _key: getBackupKey(backup),
+        name: backup.name,
+        instanceId,
+        clusterId,
+        sourceTable: backup.sourceTable,
+        expireTime: backup.expireTime,
+        startTime: backup.startTime,
+        endTime: backup.endTime,
+        sizeBytes: backup.sizeBytes,
+        state: backup.state,
+        encryptionType: backup.encryptionInfo?.encryptionType,
+        kmsKeyVersion: backup.encryptionInfo?.kmsKeyVersion,
+        webLink: getGoogleCloudConsoleWebLink(
+          `/bigtable/instances/${instanceId}/backups?project=${projectId}`,
+        ),
+      },
+    },
+  });
+}
```

Comment:
Couple of things here:

1) Use `parseTimePropertyValue()` whenever there are some timestamps
2) Could you try adding the backup resource to the Terraform? Ideally we'd want to have it be created as well but if it's tricky feel free to skip it.
3) No matter if Terraform approach worked or not, let's make sure we have a backup (e.g. you could create it manually inside the dashboard) - will populate the snapshots + could help with the next points.
4) Let's also add `BigTable Table -> HAS -> BigTable Backup`. I've noticed you've added `Cluster -> HAS -> Backup` which correct :heavy_check_mark:, but we could add this one as well. (don't forget to update tests).
5) You'll probably need to update `webLink` in here, but please re-check `webLinks` for all the resources in this PR.
6) There is some info regarding `kms`, though it doesn't necessarily mentions the name. Please try creating a backup with kms enabled, see if the kmsName is visible somewhere (it's needed for the code to be able to find kms entity) and if it is, do the similar thing like I wrote above please.

Reply:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/39617427/129125986-f815d655-5fc4-49bd-848a-79380aa35745.png">

I'm not seeing any option that would allow me to create a backup with kms enabled.

Reply:
Hmm could it be that kms gets enabled by default if the target you're creating a backup for has kms enabled already (in this case the target table/cluster)?

If that still doesn't result in kms enabled backup we may want to just wrap the code without that part for now :thinking:

Reply:
I tried creating everything again & looked for an option for `kms`. Couldn't find anything that would explicitly allow us to enable it. :(